### ISSUE TITLE
GPU display breaks under Win7 Cygwin

### ISSUE THREAD

**Issue by gaussandhisgun** [dylanaraps/neofetch#1759]
## Description

Neofetch suddenly breaks its GPU display on my Cygwin installation (Windows 7)

- [ x ] Does this issue still occur in the master branch? (**Required if issue**)

#### Neofetch version

7.1.10

## Screenshot
![изображение](https://user-images.githubusercontent.com/63512006/119372660-3989b180-bcc0-11eb-8c46-b22d16a5d4b4.png)

## Config file
Default.
## Verbose log

https://pastebin.com/raw/CjqHfC2b

**Comment by dylanaraps** [comment]
Please try the latest master.

**Comment by gaussandhisgun** [comment]
Nah, bug still occurs on latest master. Windows upgrade did not help either.
![изображение](https://user-images.githubusercontent.com/63512006/125198969-d8686e00-e26c-11eb-951e-5bb413a8822b.png)

**Comment by zerox003** [comment]
still broken on my end. (Win 11)
![ConEmu64_mSvEWMgf99](https://github.com/dylanaraps/neofetch/assets/40106969/fa77fe3f-d605-45b2-9fc6-6bc177abb891)
